### PR TITLE
Fix error on content-type response

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
Signed-off-by: Pierre-Emmanuel Jacquier <pierre-emmanuel.jacquier@epitech.eu>

### CODE REVIEW

**Comment by greut** [comment]
Thanks!

File: request_test.go [hunk 1]
```diff
@@ -14,13 +14,17 @@ import (
 	"time"
 )
 
+const (
+	jsonContentType = "application/json;charset=utf-8"
```
Comment (greut) [inline request_test.go]: `application/json` is by default `utf-8`, unlike `text/html`

Comment (pierre-emmanuelJ) [reply request_test.go]: i'll change it

**Comment by pierre-emmanuelJ** [comment]
yes 👍

**Comment by greut** [comment]
No tests :cry: ?

**Comment by pierre-emmanuelJ** [comment]
oh i forget ! i'ill do it

**Pull request merged**